miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/RealMiniAppView.kt

```diff
@@ -24,6 +24,10 @@ internal class RealMiniAppView(
                 )
             )
             settings.javaScriptEnabled = true
+            settings.allowFileAccess = true
```

Comment:
I think this should also be fine, considering that it would just enhance the experience and save us from some debugging sessions back and forth when we receive new mini apps for testing.

Reply:
I think we will need to revisit this at some point. I think there could be some security concerns here. i.e. the Mini App WebView will be able to access anything in the App's `files` directory (including other Mini App data or other data that the host App has saved in `files`). We can revisit this when we address separation of data.